tsconfig.json

```diff
@@ -1,5 +1,8 @@
 {
   "compilerOptions": {
+    "target": "ESNext",
+    "lib": ["ES2015"],
+    "moduleResolution": "node",
```

Comment:
Hey there! Can you explain these config changes to me?

Reply:
Sure thing. 

- `target` Even though Babel chooses the actual compilation target, the type checker uses this to see if the target supports certain language features. I added this target because it defaults to ES3, which doesn't support getters and setters, so TS complains if you use them.
- `lib`. The default lib includes DOM, which is incorrect and was causing a conflic with `Cache`
- `moduelResolution`. Just telling it that we're using node. It was complaining about the json resolution option without it